src/main/kotlin/app/revanced/patches/strava/fingerprints/SubscriptionFingerprint.kt

```diff
@@ -0,0 +1,12 @@
+package app.revanced.patches.strava.fingerprints
+
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import org.jf.dexlib2.Opcode
+
+object SubscriptionFingerprint : MethodFingerprint(
+    returnType = "Z",
```

Comment:
Wouldn't the fingerprint resolve to the correct method without specifying the return type?